NEWS.md

```diff
@@ -35,9 +37,6 @@
 * SQL Server: correctly enumerate schemas across databases in connections pane
   (@detule, #527).
 
-* SQL Server: now uses column type `"BIGINT"` integer64 objects 
```

Comment:
@detule I think you deleted this accidentally?

Reply:
Ugh - sorry about that! Will fix tonight, unless someone beats me to it.

Reply:
Let me do it since I'm polishing news for release.